commands/ls.go

```diff
@@ -65,6 +66,8 @@ git bug ls status:open --by creation "foo bar" baz
 		"Filter by status. Valid values are [open,closed]")
 	flags.StringSliceVarP(&options.authorQuery, "author", "a", nil,
 		"Filter by author")
+	flags.StringSliceVarP(&options.metadataQuery, "metadata", "m", nil,
```

Comment:
an example might be useful as you expect a specific format

Reply:
I've added one, though I don't want to spell out a complete GH issue; hopefully the "URL" form is clear enough.